### ISSUE TITLE
GIOU in the output

### ISSUE THREAD

**Issue by CuongNguyen218** [AlexeyAB/darknet#4182]
Hi @AlexeyAB 
I'm training my own dataset with yolov3 darknet and I see GIOU in output command. Does GIOU affect the results? I see different between iou and GIOU score. if i want to use only iou as training metric (not use Giou), how can I do it ?
![image](https://user-images.githubusercontent.com/43350150/67733244-63211f00-fa30-11e9-86a5-4370196c01e6.png)

**Comment by lsd1994** [comment]
You can look this #3249 about giuo 
And here you can change output https://github.com/AlexeyAB/darknet/blob/69b7a19f5bfce760debf2057b71b0882c535f350/src/yolo_layer.c#L426

**Comment by CuongNguyen218** [comment]
@lsd1994 ,
If i don't set iou_loss = giou, does giou affect the results?

**Comment by lsd1994** [comment]
@CuongNguyen218 
No

**Comment by CuongNguyen218** [comment]
@lsd1994 ,
Thank you.

**Comment by AlexeyAB** [comment]
To use GIoU - use these lines: https://github.com/AlexeyAB/darknet/blob/master/cfg/yolov3.coco-giou-12.cfg#L805-L806

**Comment by CuongNguyen218** [comment]
![image](https://user-images.githubusercontent.com/43350150/67775790-00f90600-fa92-11e9-8775-a1daab2a629e.png)
@AlexeyAB , I just want to only use Algo 1 instead of iou and i know algo 2 is an option in cfg file. How can I do that?
![image](https://user-images.githubusercontent.com/43350150/67775915-2be35a00-fa92-11e9-831c-65d831ac2310.png)

**Comment by IgorDavidyuk** [comment]
Hi guys, I train the csp network with coin loss, but at some point in training GIOU becomes negative and you loss explodes, that leads to nan average loss.
What do you think? Is it the dataset problem or some mistake in cfg?
